phpBB/memberlist.php

```diff
@@ -1277,7 +1277,9 @@
 			}
 
 			$param = call_user_func_array('request_var', $call);
-			$param = urlencode($key) . '=' . ((is_string($param)) ? urlencode($param) : $param);
+			// Encode strings, convert ints and bools to strings in order to prevent empty parameters.
+			// See: https://tracker.phpbb.com/browse/PHPBB3-13630
+			$param = urlencode($key) . '=' . ((is_string($param)) ? urlencode($param) : var_export($param, true));
```

Comment:
Why overcomplicate things with var_export()? This might actually cause issues due to the use of output buffering which can interfere with ob_start(). Converting things to int will also take care of the described issue.